x/ethbridge/client/cli/tx.go

```diff
@@ -59,3 +59,34 @@ func GetCmdCreateEthBridgeClaim(cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+// GetCmdBurnEth is the CLI command for burning some of your eth and triggering an event
+func GetCmdBurn(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "burn [cosmos-sender-address] [ethereum-receiver-address] [amount]",
+		Short: "burn some coins!",
+		Args:  cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+
+			txBldr := authtxb.NewTxBuilderFromCLI().WithTxEncoder(utils.GetTxEncoder(cdc))
+
+			cosmosSender, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return err
+			}
+
+			ethereumReceiver := types.NewEthereumAddress(args[1])
+
+			amount, err := sdk.ParseCoins(args[2])
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgBurn(cosmosSender, ethereumReceiver, amount)
```

Comment:
validate basic

Reply:
Might make more sense for these changes to be in https://github.com/cosmos/peggy/pull/79,  as it will introduce conflicts such as the one resolved in https://github.com/cosmos/peggy/pull/77/commits/e34c38433c1979ae1a8aeb427e87680008370e6e. @musnit can you confirm?